master/getting-started/kubernetes/tutorials/using-calicoctl.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Using calicoctl in Kubernetes
+---
+
+## Installing Calicoctl
+
+There are two ways to run `calicoctl` in Kubernetes:
+
+- Manual Install
```

Comment:
Manual Install isn't the right wording for this - I think this is "As a standalone binary" or similar.